Documentation/howto/getting_started.md

```diff
@@ -44,14 +44,14 @@ At this point, you should either have built Clair from source or have pulled the
 
 *You may need to configure [docker](https://docs.docker.com/network/) or [podman](https://podman.io/getting-started/network.html) networking if you are utilizing containers. This is out of scope for this how-to.*
 
-A basic config for combo mode can be found [here](https://github.com/quay/clair/blob/development-4.0/config.yaml.sample). Make sure to edit this config with your database settings and set "migrations" to `true` for all mode stanzas. In this basic combo mode, all "connstring" fields should point to the same database and any *_addr fields are simply ignored. For more details see the [config reference](../reference/config.md) and [deployment models](./deployment.md)
+A basic config for combo mode can be found [here](../../config.yaml.sample). Make sure to edit this config with your database settings and set "migrations" to `true` for all mode stanzas. In this basic combo mode, all "connstring" fields should point to the same database and any *_addr fields are simply ignored. For more details see the [config reference](../reference/config.md) and [deployment models](./deployment.md)
```

Comment:
@jan-zmeskal I do not think this will work. We render documentation with "mdbook" and I think any referenced links must belong within the Documentation folder. 

I could be wrong about this, you can actually test if it works by running "mdbook serve" from the root of the repository. You will need to install the mdbook tool. https://github.com/rust-lang/mdBook

Reply:
@ldelossa You are right. I replaced the relative path with web link pointing to `main` branch. I tried this locally by running `mdbook serve` and it worked fine.